contracts/identitytreestore/IdentityTreeStore.sol

```diff
@@ -149,14 +149,14 @@ contract IdentityTreeStore is Initializable, IOnchainCredentialStatusResolver, I
         return status;
     }
 
-    function _getProof(uint256 root, uint256 index) internal view returns (Proof memory) {
+    function _getProof(uint256 root, uint256 index) internal view returns (MerkleTreeProof memory) {
         uint256[] memory siblings = new uint256[](MAX_SMT_DEPTH);
         // Solidity does not guarantee that memory vars are zeroed out
         for (uint256 i = 0; i < MAX_SMT_DEPTH; i++) {
             siblings[i] = 0;
         }
 
-        Proof memory proof = Proof({
+        MerkleTreeProof memory proof = MerkleTreeProof({
```

Comment:
I think in this way we break ABI which can be a problem for clients

Reply:
reverted